README.md

```diff
@@ -179,44 +179,127 @@ Forks of this project demonstrate how to extend and integrate with other librari
 ├── package.json               <- dependencies of the project
 ├── protractor.conf.js         <- e2e tests configuration
 ├── src                        <- source code of the application
-│   ├── home
-│   │   └── components
-│   ├── index.html
-│   ├── main.ts
-│   ├── shared
-│   │   └── services
-│   │       ├── name-list...
-│   │       └── name-list...
-│   └── sw.js                  <- sample service worker
+│   └── client
+│       ├── app
+│       │   ├── +about
+│       │   │   ├── components
```

Comment:
This reminds me...we have one more violation of the style guide here. Instead of grouping by type it'll be better to group by feature, i.e. we can remove the `components` directory here, same for `+home`.

For `app`, we should move `app.component.*` outside the `components` directory and move `toolbar`, `navbar`, into separate directories within a directory called `shared`. After that we need to add barrels in each lazy-loading boundary.